demos/common/python/stream_client.py

```diff
@@ -0,0 +1,130 @@
+#
+# Copyright (c) 2023 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import cv2
+from functools import partial
+import queue
+import threading
+import tritonclient.grpc as grpcclient
+import subprocess #nosec
+from enum import Enum
+import numpy as np
+
+class StreamClient:
```

Comment:
this class might be reused in many examples but it doesn't have any documentation or even comments